include/invert_quda.h

```diff
@@ -720,6 +720,7 @@ namespace quda {
   };
 
   struct DeflationParam;//Forward declaration
+  typedef cudaColorSpinorField cudaColorSpinorFieldSet;
```

Comment:
It seems this typedef is just used to make it more obvious that this spinor contains multiple components. Or is there more to it?